src/messaging/ExchangeContext.h

```diff
@@ -199,6 +182,24 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
     void Free();
     void Reset();
 
+    /**
+     *  Determine whether a response is expected for messages sent over
+     *  this exchange.
```

Comment:
So this is a value associated with the _exchange_, right?

Reply:
Yes, only one 'response expected' message can be outstanding at a time.